docs/management/security/_index.md

```diff
@@ -159,19 +159,19 @@ While it would be a strange use case, the application should avoid composing the
 
 ## Code security
 
-In a classical Redis setup, clients are allowed full access to the command set,
+In a classical Valkey setup, clients are allowed full access to the command set,
 but accessing the instance should never result in the ability to control the
-system where Redis is running.
+system where Valkey is running.
 
-Internally, Redis uses all the well-known practices for writing secure code to
+Internally, Valkey uses all the well-known practices for writing secure code to
 prevent buffer overflows, format bugs, and other memory corruption issues.
 However, the ability to control the server configuration using the **CONFIG**
 command allows the client to change the working directory of the program and
-the name of the dump file. This allows clients to write RDB Redis files
-to random paths. This is [a security issue](http://antirez.com/news/96) that may lead to the ability to compromise the system and/or run untrusted code as the same user as Redis is running.
+the name of the dump file. This allows clients to write RDB Valkey files
+to random paths. This is [a security issue](http://antirez.com/news/96) that may lead to the ability to compromise the system and/or run untrusted code as the same user as Valkey is running.
 
-Redis does not require root privileges to run. It is recommended to
-run it as an unprivileged *redis* user that is only used for this purpose.
+Valkey does not require root privileges to run. It is recommended to
+run it as an unprivileged *valkey* user that is only used for this purpose.
 
 ## GPG key
```

Comment:
Let's delete this GPG key. If anyone sends us emails with this one, we can't decrypt them.

Reply:
done